src/MagnumPlugins/UfbxImporter/UfbxImporter.conf

```diff
@@ -1,3 +1,59 @@
 depends=AnyImageImporter
 provides=FbxImporter
 provides=ObjImporter
+
+# [configuration_]
+[configuration]
+
+# Generate normals if missing.
+generateMissingNormals=false
+
+# Fail if the file is slightly malformed.
+strict=false
+
+# Disable any exporter-specific quirks in the importer.
+disableQuirks=false
+
+# Attempt to load further files referenced by this file.
+# On .obj files this would include .mtl files.
+loadExternalFiles=false
```

Comment:
Thinking about [what you said about loading external files](https://github.com/mosra/magnum-plugins/pull/133#discussion_r1027300983) again -- I'm thinking the plugin should behave in the least surprising way by default, and loading OBJs without materials feels rather surprising to me. OTOH loading of external images won't be affected by this option as far as I can see.

Not sure about the FBX geometry caches you mentioned, how essential are they to how the model looks. Do they store morph animations or baked cloth etc. simulation?

Since this option would work only partially anyway, I'm thinking about just removing it altogether to avoid confusion, and passing `true` to ufbx always. 

Users that want control over what data are loaded and from where can supply [file loading callbacks](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1AbstractImporter.html#Trade-AbstractImporter-usage-callbacks) instead. For images that's already propagated correctly in the inherited `setupOrReuseImporterForImage()` function but for ufbx itself you still need to implement that by overriding `doSetFileCallback()` similarly to [how it's done in case of Assimp](https://github.com/mosra/magnum-plugins/blob/b5e586d45dd43a450fa54295b8c4e24137998468/src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp#L345-L360) (but again, Assimp has a lot of unnecessary complexity there with the interfaces and subclasses, with the interface you have in ufbx it'll be 10x less code).

Reply:
Yes geometry caches are usually baked simulations and such, morph targets are stored directly in the file. We probably don't need them so I'll need to think of a nicer interface for ufbx to forbid those but still load .mtl files by default. It's always nice to see how the API could be improved when doing integrations!

Reply:
I'm thinking of something like an explicit function for loading cache files -- say, the user sees in some metadata bit that the file references external caches, and in order to access those, has to call an explicit API to "pull in the gigabytes". 

While the lightweight external files such as `mtl`s are loaded implicitly.

Reply:
The cache files are actually relatively lightweight during loading as it only gathers `ufbx_cache_frame` metadata structs that refer to file offsets:

```c
typedef struct ufbx_cache_frame {
	// [abridged]
	ufbx_string filename;
	ufbx_cache_data_format data_format;     // < Format of the data in the file
	ufbx_cache_data_encoding data_encoding; // < Binary encoding of the data
	uint64_t data_offset;                   // < Byte offset into the file
} ufbx_cache_frame;
```

Though there's not much reason for supporting those if not exposed through the `Trade` interface (and geometry cache support adds some kilobytes to the library as well). If using ufbx directly you would use an utility function like `ufbx_read_geometry_cache_vec3()` to retrieve the actual data.

Reply:
So, if they're this lightweight, it wouldn't hurt much to just have the option enabled always, right? I was afraid that it'd cause literal gigabytes to get loaded and then immediately discarded because they aren't exposable through my interfaces in any way.

Reply:
It *might* cause seeking through gigabyte sized files and might cause issues with file loading callbacks as we don't have stream support in there. Actually that makes me think, maybe I should have a way to return a data blob from a file callback, instead of requiring users to make their own "virtual reader on a stream" abstractions.

Reply:
In an attempt to reduce the initial scope of this PR from growing indefinitely (as even just the tests will still take quite some time), I think I'm fine with the initial version having a caveat of "may unnecessarily seek through gigabytes if you give it gigabyte-sized cache files on input", in exchange to have `*.mtl` files loaded implicitly.

Something that's fine accounting for in a subsequent PR. It's *Assimp* we're upgrading from after all, let's not forget that ;)

Reply:
I ended up adding rudimentary ignoring for geometry cache files as I had to rework the ufbx file opening API for adding `ufbx_load_memory()`, still not 100% happy with the API but at least it doesn't require users writing an adapter for reading over a stream.